ask-sdk-servlet-support/src/com/amazon/ask/servlet/verifiers/SkillRequestTimestampVerifier.java

```diff
@@ -55,6 +58,18 @@ public SkillRequestTimestampVerifier(long toleranceInMilliseconds) {
         this.toleranceInMilliseconds = toleranceInMilliseconds;
     }
 
+    /**
+     * Constructs a new timestamp verifier with the provided tolerance and timeUnit.
+     *
+     * @params tolerance, timeUnit
+     *            the tolerance of this verifier must be non-negative and less than
+     *            {@value ServletConstants#MAXIMUM_TOLERANCE_MILLIS} after converting to milliseconds.
+     *            timeUnit must be non-null.
+     */
+    public SkillRequestTimestampVerifier(long tolerance, TimeUnit timeUnit) {
+        this(ValidationUtils.assertNotNull(timeUnit, "timeUnit").toMillis(tolerance));
```

Comment:
Just one question. The SkillRequestTimestampVerifier constructor is called from the SkillServlet class: `defaultVerifiers.add(new SkillRequestTimestampVerifier(timestampToleranceProperty != null ? timestampToleranceProperty : DEFAULT_TOLERANCE_MILLIS));`

Therefore the old constructor will be called, or do you expect the developer to pass the verifiers in the SkillServlet constructor? Nevertheless the documentation should be adjusted here: https://developer.amazon.com/de/docs/custom-skills/host-a-custom-skill-as-a-web-service.html and here: https://github.com/alexa/skill-samples-java/blob/master/tidepooler/pom.xml

Reply:
The existing constructor behavior can't be changed for backwards compatibility reasons, so effectively to opt in to the new behavior one would need to pass an instance of the verifier to the servlet. Totally agree that the docs need to be updated as they still reference some v1 concepts.

Reply:
OK I thought of that, thanks for the explanation 👍